Language_Parser/actionVerbs.py

```diff
@@ -329,22 +329,82 @@ def use(info):
     print("Using...")
 
 
-def goHelper(roomInfo, currentRoom):
+def goStairsHelper(roomTarget, currentRoomName):
+    """
+    Helper function returns the destination room name when the player
+    only specifies a form of stairs as a destination. If the current
+    room has no stairs, then returns None.
+    """
+    stairsDict = {
+        "upperHall": {"upstairs": None, "downstairs": "lowerHall"},
+        "lowerHall": {"upstairs": "upperHall", "downstairs": "basement"},
+        "basement": {"upstairs": "lowerHall", "downstairs": None}
+    }
```

Comment:
Have you considered adding attic to this? I know it doesn't technically utilize a staircase, but it might be fit here.

Reply:
The thought did cross my mind! I'll add and commit.